packages/toolpad-studio/src/server/toolpadAppBuilder.ts

```diff
@@ -270,6 +270,14 @@ if (import.meta.hot) {
               : // load compiled
                 path.resolve(currentDirectory, '../exports'),
           },
+          {
+            find: '@toolpad/core',
```

Comment:
I had to add an alias here to get `@toolpad/core` imports to work in studio, not sure why it's not needed for `@toolpad/studio-components` for example, as those just worked?

Reply:
pnpm installs the core `build` folder into studio node modules because of its package.json `publishConfig.directory` setting. During the core build command a package.json is generated in that build folder. the build folder is what gets published on npm.

The dev command of builds the sources into the build folder, but it doesn't copy the package.json. That's why module resolution doesn't work. You made it work by alisaing to the core main folder, which does have a package.json.

I added some commits that just call the copy command in the dev command. It's not perfect, it won't watch for changes to package.json, but at least it unblocks us in the PR.